src/components/pages/Analytics/Analytics.css

```diff
@@ -0,0 +1,15 @@
+h1 {
```

Comment:
Make sure to wrap this in a class specific selector to avoid style bleeding into other components. (ex `.analytics h1 {}`)